src/Components/MembersHome/Activities/activities.module.css

```diff
@@ -0,0 +1,62 @@
+* {
+  margin: 0;
+}
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-size: 25px;
+  color: white;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  background-image: url('../../../../public/assets/images/admins-background.jpg');
+  font-weight: bold;
+}
+
+.header {
+  margin: 25px 0;
+}
+
+.body {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: flex-start;
+  justify-content: space-evenly;
+  width: 95vw;
+  margin-bottom: 75px;
+}
+
+.activity {
+  border-radius: 13px;
+  box-shadow: 0px 14px 7px 1px rgb(0 0 0 / 75%);
+  background-color: #474885;
+  width: 20vw;
```

Comment:
Please try to replace the width and height with:
```
height: 400px;
width: 15vw;
```

Reply:
Add the margin bottom because when there are too many cards they stick together

Reply:
Done